spec/acceptance/sessions_spec.rb

```diff
@@ -14,14 +17,12 @@
     parameter :google_user_id, "google_user_id of user"
     context '200' do
       before do 
-        organization = FactoryBot.create(:organization, name: 'Josh', domain: ["josh.com"])
```

Comment:
why is this code block deleted

Reply:
I updated the Factory for user here, so for test case to run I am required to use the same organization as in factory, so I deleted this one and added @organization in before do block after resources